Automattic/newspack-plugin
# feat(ia): newspack primary hex color

### All Submissions:


### Changes proposed in this Pull Request:

* Updates all remaining, style, instances of Newspack legacy blue (`#3366ff`, `#36f`) to new blue (`#2240d5`). 
* Refactored all  `$primary-500` scss variable usages to use `var(--newspack-ui-color-primary)`.
* Refactored css variables starting with `--np-` to `--newspack-wizard-` to better align with `src/newspack-ui` naming convention.
* Refactored, removed unused/unnecessary css variables

A few elements affected by the changes in this PR.
| Before:      | After:      |
| ------------- | ------------- |
| ![Screenshot 2024-12-18 at 13 22 42](https://github.com/user-attachments/assets/d4bc353f-a5c6-46f5-8620-658413bc6dee) | ![Screenshot 2024-12-18 at 13 24 11](https://github.com/user-attachments/assets/64038575-9341-450d-8032-4def6ea95dd9) |
| ![Screenshot 2024-12-18 at 13 24 54](https://github.com/user-attachments/assets/3cb2e1e6-906c-42bd-a09c-406050b6b6f2) | ![Screenshot 2024-12-18 at 13 25 24](https://github.com/user-attachments/assets/29943e60-3792-4cf3-8887-437eeb6a73cb) |
| ![Screenshot 2024-12-18 at 13 27 24](https://github.com/user-attachments/assets/3f1d9b7a-da28-47c7-b3df-6d311a05cf57) | ![Screenshot 2024-12-18 at 13 26 27](https://github.com/user-attachments/assets/42dfaefd-a47b-4076-93bd-76a368b826ac) |
| ![Screenshot 2024-12-18 at 13 27 49](https://github.com/user-attachments/assets/e321cfe5-631b-48ea-b731-570bc0b3b50a) | ![Screenshot 2024-12-18 at 13 26 58](https://github.com/user-attachments/assets/e54f3976-8f8c-4666-a989-2d3a9d490ca4) |

### How to test the changes in this Pull Request:

1. Checkout this branch and build assets `git checkout origin/feat/ia-np-blue-hexcode && npm run build`
2. Navigate to _/wp-admin/admin.php?page=newspack-components-demo_ and confirm UI primary color change.
3. Navigate through wizards and confirm UI primary color change.

### Other information:
